samples/sample_vpp/src/sample_vpp_utils.cpp

```diff
@@ -729,7 +729,7 @@ mfxStatus InitMemoryAllocator(
 
         if(MFX_IMPL_HARDWARE == MFX_IMPL_BASETYPE(impl))
         {
-            pAllocator->pDevice = CreateVAAPIDevice();
+            pAllocator->pDevice = CreateVAAPIDevice(MFX_LIBVA_DRM, pInParams->strDevicePath.c_str());
```

Comment:
same

Reply:
It seems we always explicitly specify device path. Consider to reorder arguments: this will hide MFX_LIBVA_DRM from @dmitryermilov's eyes:).

Reply:
Args reordered.